x-pack/plugins/security_solution/server/lib/telemetry/tasks/detection_rule.ts

```diff
@@ -25,11 +25,11 @@ export function createTelemetryDetectionRuleListsTaskConfig(maxTelemetryBatch: n
     timeout: '10m',
     version: '1.0.0',
     runTask: async (
-      taskId: string,
+      _taskId: string,
       logger: Logger,
       receiver: ITelemetryReceiver,
       sender: ITelemetryEventsSender,
-      taskExecutionPeriod: TaskExecutionPeriod
+      _taskExecutionPeriod: TaskExecutionPeriod
```

Comment:
Yes, it was to avoid an annoying warning :)
I'll change the query to use these values, but for the first execution `taskExecutionPeriod.last` is undefined, then we need a fallback, eg, we should query either from `taskExecutionPeriod.last` until `taskExecutionPeriod.current` OR the last three hours
WDYT?